devito/ops/transformer.py

```diff
@@ -186,7 +195,7 @@ def create_ops_dat(f, name_to_ops_dat, block):
                 Symbol(base.name),
                 Symbol(d_m.name),
                 Symbol(d_p.name),
-                Byref(f.indexify([0])),
+                f.indexify([0]),
```

Comment:
The `ops_decl_dat` is documented in OPS API as:
```
ops_dat ops_decl_dat(ops_block block, int dim, int* size, int *base,int *dm, int *dp, T *data, char *type, char *name) 
...
data     ->     input data of typeT
...
```

So, I need a pointer to the data. 
This line represents a `Function` access in Devito and it is translated into an array in C. 

Actually, after i removed the `Byref` I should have dropped the indexify, and just pass the `f`.

I hope that this will be translated into an access to the function (for example if the function was declared as `v = Function(name='v', grid=grid, space_order=2)`, I want this part to just generate `v` in C or, if this is not possible, generate as it was being done `&(v[0])`).